examples/holistic_landmarker/android/app/src/main/java/com/google/mediapipe/examples/holisticlandmarker/fragments/CameraFragment.kt

```diff
@@ -0,0 +1,387 @@
+        )
+    }
+
+    // Declare and bind preview, capture and analysis use cases
+    @SuppressLint("UnsafeOptInUsageError")
+    private fun bindCameraUseCases() {
+
+        // CameraProvider
+        val cameraProvider = cameraProvider
+            ?: throw IllegalStateException("Camera initialization failed.")
+
+        val cameraSelector =
+            CameraSelector.Builder().requireLensFacing(cameraFacing).build()
+
+        // Preview. Only using the 4:3 ratio because this is the closest to our models
+        preview = Preview.Builder().setTargetAspectRatio(AspectRatio.RATIO_4_3)
+            .setTargetRotation(fragmentCameraBinding.viewFinder.display.rotation)
+            .build()
+
+        // ImageAnalysis. Using RGBA 8888 to match how our models work
+        imageAnalyzer =
+            ImageAnalysis.Builder().setTargetAspectRatio(AspectRatio.RATIO_4_3)
+                .setTargetRotation(fragmentCameraBinding.viewFinder.display.rotation)
+                .setBackpressureStrategy(ImageAnalysis.STRATEGY_KEEP_ONLY_LATEST)
+                .setOutputImageFormat(ImageAnalysis.OUTPUT_IMAGE_FORMAT_RGBA_8888)
+                .build()
+                // The analyzer can then be assigned to the instance
+                .also {
+                    it.setAnalyzer(backgroundExecutor) { image ->
+                        detectFace(image)
+                    }
+                }
+
+        // Must unbind the use-cases before rebinding them
+        cameraProvider.unbindAll()
+
+        try {
+            // A variable number of use-cases can be passed here -
+            // camera provides access to CameraControl & CameraInfo
+            camera = cameraProvider.bindToLifecycle(
+                this, cameraSelector, preview, imageAnalyzer
+            )
+
+            // Attach the viewfinder's surface provider to preview use case
+            preview?.setSurfaceProvider(fragmentCameraBinding.viewFinder.surfaceProvider)
+        } catch (exc: Exception) {
+            Log.e(TAG, "Use case binding failed", exc)
+        }
+    }
+
+    private fun detectFace(imageProxy: ImageProxy) {
+        holisticLandmarkerHelper.detectLiveStreamCamera(
+            imageProxy = imageProxy,
+            isFrontCamera = cameraFacing == CameraSelector.LENS_FACING_FRONT
+        )
+    }
+
+    override fun onConfigurationChanged(newConfig: Configuration) {
+        super.onConfigurationChanged(newConfig)
+        imageAnalyzer?.targetRotation =
+            fragmentCameraBinding.viewFinder.display.rotation
+    }
+
+    @SuppressLint("NotifyDataSetChanged")
+    override fun onError(error: String, errorCode: Int) {
+        Log.e(TAG, "An error $error, code $errorCode occurred")
+        activity?.runOnUiThread {
+            faceBlendshapesResultAdapter.updateResults(null)
+            faceBlendshapesResultAdapter.notifyDataSetChanged()
+        }
+    }
+
+    // Update UI after face have been detected. Extracts original
```

Comment:
Comment is a copy/paste from face landmarker. Should update to reflect what actually triggers the action.